design/design-doc-composition-functions.md

```diff
@@ -1,8 +1,8 @@
- # Composition Functions
+ # Composition Functions: Beta
 
-* Owners: Nic Cope (@negz), Sergen Yalçın (@sergenyalcin)
-* Reviewers: Crossplane Maintainers
-* Status: Draft
+* Owners: Nic Cope (@negz)
+* Reviewers: Hasan Turken (@turkenh), Jared Watts (@jbw976)
```

Comment:
I chose these reviewers because:

* @turkenh has context from reviewing the original, alpha design
* @jbw976 will be leading a new developer experience effort, which I feel relates closely to Functions